src/migrations/20211209211641-use-walletid-for-ledger.ts

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  async up(db) {
+    {
+      const result = await db.collection("users").updateMany({}, [
```

Comment:
I thought this PR should be non-risk ie. without a migration?

Reply:
this is a migration just to copy `walletPublicId` to `walletId` in the user collection. this is not the risky part. 

the "risky" part is related to the update of the Ledger